test/test_read_concern.py

```diff
@@ -118,8 +118,14 @@ def test_aggregate_out(self):
         except OperationFailure:
             # "ns doesn't exist"
             pass
-        self.assertNotIn('readConcern',
-                         self.listener.results['started'][0].command)
+
+        # Aggregate with $out supports readConcern MongoDB 4.2 onwards.
+        if client_context.version >= Version(4, 1):
```

Comment:
I think you can just do `client_context.version >= (4, 1)` and avoid importing or using Version.